Source/WebCore/platform/graphics/FontCascadeFonts.h

```diff
@@ -100,8 +101,8 @@ class FontCascadeFonts : public RefCounted<FontCascadeFonts> {
         std::unique_ptr<MixedFontGlyphPage> m_mixedFont;
     };
 
-    GlyphPageCacheEntry m_cachedPageZero;
-    HashMap<int, GlyphPageCacheEntry> m_cachedPages;
+    EnumeratedArray<ResolvedEmojiPolicy, GlyphPageCacheEntry, ResolvedEmojiPolicy::RequireEmoji> m_cachedPageZero;
```

Comment:
There is no reason to single out the zero page any more. It should just be part of the `HashMap`.